### DISCUSSION TITLE
Как подключать отдельные функции/модули в тотум

### DISCUSSION THREAD

**Discussion by Nicolgeh** [totumonline/totum-ru-issues-and-discussions#329] [Как это сделать?] [answered]
Делема такая: Не совсем ( от слова совсем ) понятно как происходит подключение новых функций в тотум. Т.е. написал свою функцию на PHP для тотума, закинул в нужные папки ( указанные в документации ), но от тотума ноль ответа. Как можно использовать написанные свои методы в тотуме? И как общаться с тотумом из PHP кода ( обращаться к таблицам, менять значения в полях )

**Comment by ttm-alexeypolunin** [comment, marked as answer]
По этой инструкции люди делают: https://ru.docs.totum.online/function-extension

О чем мы забыли там написать — название функции **обязательно должно начинаться** с `ext*`, например `extHello `.

Относительно взаимодействия через PHP: https://ru.docs.totum.online/additional-interfaces

Больше мы ничем вам помочь не можем.
